apiserver/facades/agent/logger/logger_test.go

```diff
@@ -81,11 +154,13 @@ func (s *loggerSuite) TestWatchLoggingConfigNothing(c *gc.C) {
 }
 
 func (s *loggerSuite) setLoggingConfig(c *gc.C, loggingConfig string) {
-	err := s.Model.UpdateModelConfig(map[string]interface{}{"logging-config": loggingConfig}, nil)
-	c.Assert(err, jc.ErrorIsNil)
-	modelConfig, err := s.Model.ModelConfig()
-	c.Assert(err, jc.ErrorIsNil)
-	c.Assert(modelConfig.LoggingConfig(), gc.Equals, loggingConfig)
+	s.change.Config["logging-config"] = loggingConfig
```

Comment:
4 times for this pattern including `SetUpTest`. I'd make a method accepting the auth to produce a `LoggerAPI`.